icu4c/source/test/cintltst/cdateintervalformattest.c

```diff
@@ -422,7 +426,7 @@ static void TestFormatCalendarToResult() {
             UPRV_LENGTHOF(expectedFieldPositions));
     }
     {
-        const char* message = "Field position test 1";
+        const char* message = "Field position test 2";
```

Comment:
Make this one 3 and the next one 4

Reply:
why?

Reply:
Nevermind, I saw another change but that was in the previous function.